### PR TITLE
Feature/aws iam user

### LINKED ISSUE #53: [Bug]: Crash b/c 1Password is locked
### What happened?

Lisa crashed because 1Password was locked 

### Lisa CLI status

 A summary of your Lisa setup

 ️💁 Your path is: /Users/liss/dev/sites
 ️💁 Versions of your packages:
   Lisa CLI: 2.12.0
   1Password CLI: 2.21.0
   Ansible: 2.15.4
   AWS CLI: 2.13.21
   GitHub CLI: 2.35.0
   Node: 20.7.0
   Trellis CLI: 1.11.1
   Valet: 3.3.2
   Vercel: 29.3.0
   WP CLI: 2.8.1
   Composer: 2.6.3
   PHP: 8.1.23

### Relevant log output

```shell
❯ lisa kinsta create

⚡️ Creating Kinsta site!

node:internal/errors:865
  const err = new Error(message);
              ^

Error: Command failed: op item get l2i57yslyjfr5jsieew4imwxgq --fields label="kinsta.api key",label="kinsta.company id"
[ERROR] 2023/09/29 09:31:22 error initializing client: 1Password app is locked. Please open 1Password, unlock it with your password, and then try again

    at ChildProcess.exithandler (node:child_process:422:12)
    at ChildProcess.emit (node:events:514:28)
    at maybeClose (node:internal/child_process:1105:16)
    at ChildProcess._handle.onexit (node:internal/child_process:305:5) {
  code: 1,
  killed: false,
  signal: null,
  cmd: 'op item get l2i57yslyjfr5jsieew4imwxgq --fields label="kinsta.api key",label="kinsta.company id"',
  stdout: '',
  stderr: '[ERROR] 2023/09/29 09:31:22 error initializing client: 1Password app is locked. Please open 1Password, unlock it with your password, and then try again\n'
}

Node.js v20.7.0
```

### PR DESCRIPTION
**Pull request by lissmeister**
Create IAM user
Save AWS access key and secret for user in 1password
Update Bucket Public Policy to allow the newly created user to read & write to the bucket
Set Bucket LifeCyclerRule

closes #53

### CODE REVIEW

File: tasks/services/aws.js [hunk 1]
```diff
@@ -2,40 +2,86 @@ import {
...
+    )
+
+    await s3Client.send(
+      new PutBucketLifecycleConfigurationCommand({
+        Bucket: bucketName,
+        ExpectedBucketOwner: accountId,
+        LifecycleConfiguration: {
+          Rules: [
+            {
+              Expiration: { ExpiredObjectDeleteMarker: true },
```
Comment (chrillep) [inline tasks/services/aws.js:75]: 30 dagar ?

**Review by chrillep — CHANGES_REQUESTED** [review]

File: tasks/services/aws.js (hunk 1 above)
Comment (lissmeister) [reply tasks/services/aws.js]: Är som det ska va

**Comment by chrillep** [comment]
Woop Woop

**Review by chrillep — APPROVED** [review]
Woop woop

**Pull request merged**